src/index.ts

```diff
@@ -202,7 +205,7 @@ export function replaceTscAliasPaths(
     return orig;
   };
 
-  const replaceAlias = (file: string): boolean => {
+  const replaceAlias = (file: string, resolveFullPath?: boolean): boolean => {
```

Comment:
Since fullpath resolution is an overlapping problem with alias resolution, I felt like it was reasonable to add it here. It feels a bit like a side effect, but a rename of the function (e.g. to `resolveImporPaths`) would make it less weird.

I considered doing it separately from this function, but that would require an additional save to disk. Probably wouldn't really matter.